src/databricks/labs/ucx/assessment/jobs.py

```diff
@@ -106,11 +108,14 @@ def _prepare(all_jobs) -> tuple[dict[int, set[str]], dict[int, JobInfo]]:
             if not job.job_id:
                 continue
             job_assessment[job.job_id] = set()
-            if not job.creator_user_name:
+            creator_user_name = job.creator_user_name
+            if not creator_user_name:
                 logger.warning(
                     f"Job {job.job_id} have Unknown creator, it means that the original creator has been deleted "
                     f"and should be re-created"
                 )
+                # Normalization.
+                creator_user_name = None
```

Comment:
is it when `job.creator_user_name == ""`?

Reply:
Indeed. The existing code treated `""` and `None` as equivalent. This was intended to ensure we only end up with `None`.

I like the `or None` suggestion above better.